web/server/next.config.ts

```diff
@@ -56,7 +56,7 @@ export default withWorkers(
       });
 
       config.module.rules.push({
-        test: /\.(webp|svg|mp3|wav)$/,
+        test: /\.(webp|mp3|wav)$/,
         exclude: /node_modules/,
         use: [
           {
```

Comment:
Is there a particular reason why we use the `url-loader` for SVG files, currently? Here, I remove SVG files from this rule because otherwise, I can't url-import SVG files in CSS `background-image` properties.

Reply:
At some point in time I tried to use SVGs directly at seabattle, but latter I gave up and wrapped them in react components, so I think this is fine